pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -28,10 +29,11 @@ import (
 const defaultOVSDBFile = "db.sock"
 
 type OVSBridge struct {
-	ovsdb        *ovsdb.OVSDB
-	name         string
-	datapathType string
-	uuid         string
+	ovsdb             *ovsdb.OVSDB
+	name              string
+	datapathType      string
+	uuid              string
+	isHardwareOffload bool
```

Comment:
could we rename this flag to `isHardwareOffloadEnabled` (along with the corresponding method in the `OVSBridgeClient` interface)?

Reply:
done